docker/daemon/daemon_dest.go

```diff
@@ -95,7 +95,7 @@ func (d *daemonImageDestination) Close() {
 	if !d.committed {
 		logrus.Debugf("docker-daemon: Closing tar stream to abort loading")
 		// In principle, goroutineCancel() should abort the HTTP request and stop the process from continuing.
-		// In practice, though, https://github.com/docker/engine-api/blob/master/client/transport/cancellable/cancellable.go
+		// In practice, though, https://github.com/docker/api/docker/types/blob/master/client/transport/cancellable/cancellable.go
```

Comment:
That URL does not exist. AFAICS the link should now go to https://github.com/golang/net/blob/master/context/ctxhttp/ctxhttp_pre17.go , _and_ the comment should say that this path is only used with Go < 1.7. It seems that ≥1.7, where `net/http` has a native `context` support, can actually abort sending the request in _some_ cases, but AFAICS not in the “write request” phase (again, it would continue sending the data in the background, only return to the caller early: https://golang.org/src/net/http/transport.go `roundTrip` sends the request to `pc.writech`, and I can’t see that `CancelRequest` aborts the anything in the `writeLoop` goroutine).